shopfloor_packing_info/views/shopfloor_packing_info_views.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="shopfloor_packing_info_tree_view" model="ir.ui.view">
+        <field name="name">shopfloor.packing.info tree</field>
+        <field name="model">shopfloor.packing.info</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="text" />
+                <field name="active" />
+            </tree>
+        </field>
+    </record>
+    <record id="shopfloor_packing_info_form_view" model="ir.ui.view">
+        <field name="name">shopfloor.packing.info form</field>
+        <field name="model">shopfloor.packing.info</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <field name="active" invisible="1" />
+                    <button
+                        type="object"
+                        name="toggle_active"
+                        string="Archive"
+                        attrs="{'invisible': [('active', '=', False)]}"
+                    />
+                    <button
+                        type="object"
+                        name="toggle_active"
+                        string="Restore"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+                </header>
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name" />
+                            <field name="text" />
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="shopfloor_packing_info_search_view" model="ir.ui.view">
+        <field name="name">shopfloor.packing.info search</field>
+        <field name="model">shopfloor.packing.info</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name" />
+                <field name="text" />
+                <field name="active" />
+                <separator />
+                <filter
+                    string="All"
+                    name="all"
+                    domain="[('active', 'in', [False, True])]"
+                />
```

Comment:
did you find examples of this filter in core modules?

I find this:

```xml

                <filter
                    string="Archived"
                    name="inactive"
                    domain="[('active','=',False)]"
                />

```

and if I understand how the filter can be useful, but maybe we prefer UX consistency and use the `Archived` filter? (and so when you have this filter you don't need the active field in the tree view as you see either the archived either the unarchived records)

Reply:
I didn't search for any example, I just wanted the same funcionality that are in the scheduled actions by default, but here optional. I can change it by yours.